x-pack/plugins/endpoint/public/applications/endpoint/models/policy_details_config.ts

```diff
@@ -14,10 +14,10 @@ const entries = <T extends object>(o: T): Array<[keyof T, T[keyof T]]> =>
 type DeepPartial<T> = { [K in keyof T]?: DeepPartial<T[K]> };
 
 /**
- * Returns a deep copy of PolicyDetailsConfig
+ * Returns a deep copy of `UIPolicyConfig` object
```

Comment:
wew did it werk?

Reply:
Yeah - just needed to get the types right